website/docs/r/security_center_assessment.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_assessment"
+description: |-
+  Manages the Security Center Assessment for Azure Security Center.
+---
+
+# azurerm_security_center_assessment
+
+Manages the Security Center Assessment for Azure Security Center.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "example-network"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  address_space       = ["10.0.0.0/16"]
+}
+
+resource "azurerm_subnet" "internal" {
+  name                 = "internal"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefixes     = ["10.0.2.0/24"]
+}
+
+resource "azurerm_linux_virtual_machine_scale_set" "example" {
+  name                = "example-vmss"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku                 = "Standard_F2"
+  instances           = 1
+  admin_username      = "adminuser"
+
+  admin_ssh_key {
+    username   = "adminuser"
+    public_key = file("~/.ssh/id_rsa.pub")
+  }
+
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
+    version   = "latest"
+  }
+
+  os_disk {
+    storage_account_type = "Standard_LRS"
+    caching              = "ReadWrite"
+  }
+
+  network_interface {
+    name    = "example"
+    primary = true
+
+    ip_configuration {
+      name      = "internal"
+      primary   = true
+      subnet_id = azurerm_subnet.internal.id
+    }
+  }
+}
+
+resource "azurerm_security_center_assessment_metadata" "example" {
+  display_name = "Test Display Name"
+  severity     = "Medium"
+  description  = "Test Description"
+}
+
+resource "azurerm_security_center_assessment" "example" {
+  assessment_metadata_id = azurerm_security_center_assessment_metadata.example.id
+  target_resource_id     = azurerm_linux_virtual_machine_scale_set.example.id
+
+  status {
+    code = "Healthy"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `assessment_metadata_id` - (Required) The ID of the security Assessment Metadata resource. Changing this forces a new security Assessment to be created.
```

Comment:
this would then be
```suggestion
* `assessment_policy_id` - (Required) The ID of the security Assessment policy to apply to this resource. Changing this forces a new security Assessment to be created.
```